Bulldozer.java

```diff
@@ -0,0 +1,18 @@
+package org.example;
+
+public class Bulldozer extends Machine {
+

```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present